src/App.css

```diff
@@ -0,0 +1,220 @@
+  background-color: rgb(89, 143, 143);
+}
+
+.details {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 40vh;
+  cursor: pointer;
+}
+
+.nav {
+  color: white;
+}
+
+.flags {
+  width: 20vw;
+}
+
+.viewmore {
+  background-color: darkslategrey;
+  color: white;
+}
+
+.navlink {
+  color: white;
+  text-decoration: none;
+}
+
+.search-country {
+  width: 20vw;
+  height: 5vh;
+  padding-left: 1vw;
+  box-shadow: 0 0 3px gray;
+  border: none;
+  margin-right: 2vw;
+  outline: none;
+  border-radius: 5px;
+}
+
+.cont-flag {
+  width: 30vw;
+}
+
+.details-cont {
+  width: 60vw;
+  height: 80vh;
+  margin: auto;
+  margin-top: 10vh;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.cont-in {
+  margin: auto;
+}
+
+.region {
+  margin-top: 5%;
+}
+
+.arr {
+  margin-left: 5vw;
+  margin-top: 5vh;
+  font-size: 30px;
+}
+
+.details-cont-inner > div {
+  padding-top: 10px;
+  padding-bottom: 10px;
+  padding-left: 15px;
+  padding-right: 15px;
+  cursor: pointer;
+  display: flex;
+  justify-content: space-between;
+}
+
+.details-cont-inner > :nth-child(even) {
+  background-color: rgb(32, 57, 57);
+}
+
+.details-cont-inner > :nth-child(odd) {
+  background-color: rgb(89, 143, 143);
+}
+
+@media only screen and (max-width: 768px) {
+  .flags {
+    width: 30vw;
+  }
+
+  .home {
+    width: 100%;
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+    gap: 0;
+  }
+
+  .home > :nth-child(4n+1),
+  .home > :nth-child(4n) {
+    background-color: darkslategrey;
+  }
+
+  .home > :nth-child(4n+2),
+  .home > :nth-child(4n+3) {
+    background-color: rgb(89, 143, 143);
+  }
+
+  .details {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    text-align: center;
+    height: 30vh;
+    cursor: pointer;
+  }
+
+  .nav-inner {
+    display: flex;
+    align-items: center;
+    justify-content: space-around;
+    width: 45%;
+  }
+
+  .search-country {
+    width: 40vw;
+    height: 5vh;
+    padding-left: 1vw;
+    box-shadow: 0 0 3px gray;
+    border: none;
+    margin-right: 2vw;
+    outline: none;
+    border-radius: 5px;
+  }
+
+  .details-cont {
+    width: 90vw;
+    height: 80vh;
+    margin: auto;
+    margin-top: 20vh;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: space-between;
+  }
+
+  .cont-flag {
+    width: 90vw;
+  }
+
+  .cont-in {
+    margin: auto;
+  }
+
+  .nav {
+    color: white;
+  }
+
+  .details-cont-inner {
+    width: 100vw;
+  }
+
+  .details-cont-inner > div {
+    padding-top: 10px;
+    padding-bottom: 10px;
+    padding-left: 15px;
+    padding-right: 15px;
+    cursor: pointer;
+    display: flex;
+    justify-content: space-between;
+  }
+
+  .details-cont-inner > :nth-child(even) {
+    background-color: rgb(32, 57, 57);
+  }
+
+  .details-cont-inner > :nth-child(odd) {
+    background-color: rgb(89, 143, 143);
+  }
+}
```

Comment:
- [ ] `Required`
- [ ] Colors (select one main color for your website). ✔️ 
- [ ] Typography: font face, size and weight. ✔️ 
- [ ] However the  Layout: composition missed a few elements. 😞 
- [ ] kindly add the missing elements
- [ ] please sceenshoots below and the highlighted areas 👇 

`original`
![image](https://github.com/Developer-Kingz/metrics/assets/66230306/656d8985-5b64-41f6-bc86-23061fe3d266)

`your design`

![image](https://github.com/Developer-Kingz/metrics/assets/66230306/0a49ac16-5c34-43ef-8703-612048e0fed9)